Kitodo-DataAccess/src/main/java/org/kitodo/dataaccess/XMLSchema.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataaccess;
+
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.RDFNode;
+import org.kitodo.dataaccess.storage.memory.MemoryNodeReference;
+import org.kitodo.dataaccess.storage.memory.MemoryStorage;
+
+/**
+ * The XML Schema datatypes.
+ *
+ * @see "https://www.w3.org/TR/xmlschema-2/"
+ */
+public class XMLSchema {
+    /**
+     * A reference to a named linked data node that represents an integer type.
+     */
+    public static class IntegerDatatype implements NodeReference {
```

Comment:
In Coding Guideline we have pointed out that every file can contain only one class.